README.md

```diff
@@ -1,8 +1,10 @@
 # Welcome to Alaveteli!
 
-[![Build Status](https://secure.travis-ci.org/mysociety/alaveteli.png)](http://travis-ci.org/mysociety/alaveteli) [![Coverage Status](https://coveralls.io/repos/mysociety/alaveteli/badge.png?branch=develop)](https://coveralls.io/r/mysociety/alaveteli) [![Code Climate](https://codeclimate.com/github/mysociety/alaveteli.png)](https://codeclimate.com/github/mysociety/alaveteli)
+[![CI](https://img.shields.io/travis/mysociety/alaveteli?label=CI)](http://travis.org/mysociety/alaveteli)
+[![RuboCop](https://img.shields.io/github/workflow/status/mysociety/alaveteli/RuboCop?label=RuboCop)](https://github.com/mysociety/alaveteli/actions?query=workflow%3ARuboCop)
+[![Coverage Status](https://img.shields.io/coveralls/github/mysociety/alaveteli/develop)](https://coveralls.io/r/mysociety/alaveteli)
+[![Code Climate](https://img.shields.io/codeclimate/maintainability-percentage/mysociety/alaveteli)](https://codeclimate.com/github/mysociety/alaveteli)
```

Comment:
Is there a maintainability grade (as per existing) rather than percentage? Not sure a percentage makes sense for this. No big deal if not.